hack/release-notes/main.go

```diff
@@ -18,47 +18,50 @@ const (
 	noGroup  = "nogroup"
 	repoName = "elastic/cloud-on-k8s"
 
-	releaseNotesTemplate = `:issue: https://github.com/{{.Repo}}/issues/
-:pull: https://github.com/{{.Repo}}/pull/
+	releaseNotesTemplate = `## Release Notes for {{.Version}} 
```

Comment:
Heading hierarchy seems off
```suggestion
	releaseNotesTemplate = `## {{.Version}} [elastic-cloud-kubernetes-{{.Version | replace "." "" }}-release-notes] 
```
with a little template helper func